uk.co.compucorp.civicrm.hrleaveandabsences/tests/phpunit/CRM/HRLeaveAndAbsences/BAO/LeaveBalanceChangeTest.php

```diff
@@ -837,6 +833,44 @@ public function testTheTotalBalanceChangeForALeaveRequestWithoutBalanceChangesSh
     $this->assertEquals(0, LeaveBalanceChange::getTotalBalanceChangeForLeaveRequest($leaveRequest));
   }
 
+  public function testTheTotalBalanceChangeForALeaveRequestShouldBeOnlyTheExpiredAmountWhenExpiredOnlyIsTrue() {
+    $leaveRequest = LeaveRequestFabricator::fabricateWithoutValidation([
```

Comment:
please adjust to single space between the arrow operator and operand for this array.
and also for the other similar arrays in this function